apps/admin-web/src/common/Topbar/Topbar.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from 'next/link'
+import styles from './style.module.css'
+
+export default function Topbar() {
+  return (
+    <ul className={styles.navbar}>
+      <li className={styles.navbarItem}>
+        <Link href="/pendingReviews">Pending Reviews</Link>
+      </li>
+      <li>
+        <Link href="/approvedReviews">Approved Reviews</Link>
+      </li>
+      <li>
+        <Link href="/genEd">GenEd</Link>
```

Comment:
I feel a bit weird about the CamelCase route names. Should this be 

```
/review/pending
/review/approved
/genEd -> Not sure what does this page can do
```

Or, just merge 2 routes into one then use query parameters to filter the data (Suggested). Like this 

```
/reviews -> Show all reviews
/reviews?status=approved -> Show only approved reviews
/reviews?status=pending -> Show only pending reviews
```

Reply:
i agree that the route should be lowercase or kebab-case only. However, the final routes will depend on the design.